packages/amplify-frontend-flutter/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "amplify-frontend-flutter",
+  "version": "0.0.2-flutter-preview.0",
+  "description": "amplify-cli front-end plugin for Flutter projects",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws-amplify/amplify-cli.git",
+    "directory": "packages/amplify-frontend-flutter"
+  },
+  "author": "Amazon Web Services",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "keywords": [
+    "graphql",
+    "appsync",
+    "aws"
+  ],
+  "dependencies": {
+    "chalk": "^3.0.0",
```

Comment:
make sure that these dependency versions are aligned with other packages, like `inquirer` is `7.3.3` currently.

Reply:
Inquirer dependency verision is fixed; all others verified.